src/kaleidoscope/driver/led/WS2812.h

```diff
@@ -80,12 +80,12 @@ class WS2812 {
   }
 
  private:
-  Color leds_[ledCount];
+  Color leds_[ledCount]; // NOLINT(runtime/arrays)
```

Comment:
Like in the Virtual case above, this is a false positive. `ledCount` is a template argument forced into a local `constexpr`, so in practice, the size of the array is known at compile time, it isn't variadic. Since cpplint can't figure that out on its own, we ignore the warning instead.